InformationScripting/src/commands/CScript.cpp

```diff
@@ -88,6 +98,69 @@ Interaction::CommandResult* CScript::execute(Visualization::Item*, Visualization
 			qDebug() << "Error in Python: " << BoostPythonHelpers::parsePythonException();
 		}
 	}
+	else if (command == "methods")
+	{
+		auto query = AstSource::instance().createMethodQuery(node, args);
+		auto compositeQuery = new CompositeQuery();
+		compositeQuery->connectToOutput(query);
+		QueryExecutor queryExecutor(compositeQuery);
```

Comment:
In this case, do we need a composite query at all? Can't we just pass query directly?